### ISSUE TITLE
Info > Edit says EDITOR not set: NotPresent

### ISSUE THREAD

**Issue by CHithubber** [francisdb/vpxtool#270]
Getting the following error:
_____________________________________

thread 'main' panicked at src/lib.rs:1206:42:
EDITOR not set: NotPresent
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
_____________________________________
Version: vpxtool-macOS-aarch64-v0.11.3

**Comment by francisdb** [comment]
Thanks for reporting this

As a workaround you can run `EDITOR=nano vpxtool ...`
Or set the env var EDITOR in your environment (~/.bashrc or ~/.zshrc or ...)

**Comment by francisdb** [comment]
Also I should add that that as an option in the config file in case users want to use a specific editor like vscode.

**Comment by CHithubber** [comment]
> Thanks for reporting this
> 
> As a workaround you can run `EDITOR=nano vpxtool ...` Or set the env var EDITOR in your environment (~/.bashrc or ~/.zshrc or ...)

Sorry, how do I run that command in the Terminal to be able to open th einfo part og a table? Can you give an example, thanks?

**Comment by francisdb** [comment]
Have you used 'nano' before?
`EDITOR=nano vpxtool frontend`
That will open the frontend and when you select edit it will open nano.
Make your edits and save the file. Then select force reload to apply the updates in the tables cache file.

**Comment by CHithubber** [comment]
yes, nano I have uses but just for very simple tasks like 'nano /path' edit stuff, like .host or simple cfg files. Super straightforward. Have not dived into variations of usage.

Okay, I will give some more try to get through that in Terminal.👍

**Comment by francisdb** [comment]
@CHithubber see #272 for a next step where you will be able to configure a gui editor like vscode
